tests/ops/integration_tests/saas/test_delighted_task.py

```diff
@@ -0,0 +1,92 @@
+import random
+from typing import List
+
+import pytest
+import requests
+
+from fides.api.ops.graph.graph import DatasetGraph
+from fides.api.ops.models.privacy_request import PrivacyRequest
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.service.connectors import get_connector
+from fides.api.ops.task import graph_task
+from fides.api.ops.task.filter_results import filter_data_categories
+from fides.api.ops.task.graph_task import get_cached_data_for_erasures
+from fides.core.config import get_config
+from tests.ops.graph.graph_test_util import assert_rows_match
+
+CONFIG = get_config()
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_delighted
+def test_delighted_connection_test(delighted_connection_config) -> None:
+    get_connector(delighted_connection_config).test_connection()
+
+async def test_delighted_access_request_task(
+    db,
+    policy,
+    delighted_connection_config,
+    delighted_dataset_config,
+    delighted_identity_email,
+) -> None:
+    """Full access request based on the Delighted SaaS config"""
+
+    privacy_request = PrivacyRequest(
+        id=f"test_delighted_access_request_task_{random.randint(0, 1000)}"
```

Comment:
I haven't reviewed the whole PR, this spot just caught my eye. We have had flaky tests from using the random int in places like this before. Random between 0 and 1000 seems unlikely to clash, but we have seen it happen more than I would expect. Switching to a `uuid` will keep this from happening.

````suggestion
        id=f"test_delighted_access_request_task_{uuid.uuid4()}"
````

Reply:
This is boiler plate code in all our connector tests. Let me create a PR to fix this in all our tests.